package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@layerfi/components",
-  "version": "0.1.75",
+  "version": "0.1.76",
```

Comment:
This is our published version, just catching up since it hasn't been updated in `main` yet.